OrbitGl/TimeGraph.cpp

```diff
@@ -7,6 +7,7 @@
 #include <OrbitBase/Logging.h>
 
 #include <algorithm>
+#include <iomanip>
```

Comment:
Do you need that include? My guess is, it was automatically added by clangd, wasn' it?